app/views/twitter/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<% unless Rails.application.secrets.twitter['enabled'] == false %>
+  <div class=twitter>
+      <div class="twitter_header">
+        Tweets by <%= @twitter_name %></p>
+      </div>
+    <% @tweets.each do |e| %>
+      <div class=tweets>
+          <div class="tweets_pic twitter-image-frame"><%= image_tag("leap-icon.jpg") %></div>
+        <div class=tweets_name>
+          @<%= @twitter_screen_name %>
+        </div>
+        <div class=tweet_text>
+          <%= " #{e.text}" %>
+        </div>
+        <div class=text_date>
+          <%= @tweet_time %> <% t = e.created_at%> <%="#{t.strftime("%m/%d/%Y")}"%>
```

Comment:
This is a bit hard to understand for me because it does not match my expectations.
I would expect `@tweet_time` to contain a timestamp. Instead it contains the string 'created at' that is set in the controller. I would prefer to insert strings as is (see line 4 here).
In addition I think this
```erb
<%="#{t.strftime("%m/%d/%Y")}"%>
```
Is overly complex. The erb tag with `=` sign will insert the content as a string. Now the content is a string (`"..."`) into which we only insert `t.strftime...`. I think we could just do
```
<%= t.strftime(...) %>
```
instead. I might be wrong but then
```
<%= t.strftime(...).to_s %>
```
should work. :thought_balloon: